### DISCUSSION TITLE
Welcome to doctrine1 Discussions!

### DISCUSSION THREAD

**Discussion by thePanz** [LExpress/doctrine1#115] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thirsch** [comment]
Is it also possible to create a discussions-board on the organisation instead of the repository? From my perspective, issues within a repository would be enough, but it would be good to discuss general topics outside of a specific repo.

**Reply by connorhu** [reply]
I think we're here for symfony, so the main repo could be symfony1, which is where the org discussion fits.

**Reply by thePanz** [reply]
Done! ✅
